node/api/hostdb.go

```diff
@@ -202,7 +205,7 @@ func (api *API) hostdbFilterModeHandlerPOST(w http.ResponseWriter, req *http.Req
 	}
 
 	// Set list mode
-	if err := api.renter.SetFilterMode(fm, params.Hosts); err != nil {
```

Comment:
We should improve this API with add and delete endpoints instead of setting the list with each request.